src/pages/main/processor/helpers.ts

```diff
@@ -68,6 +68,7 @@ export const getNeedToExpandNodeUids = (
   const _expandedItems: TNodeUid[] = [];
   selectedNodeUids.map((uid) => {
     let node = validNodeTree[uid];
+    if (!node) return;
```

Comment:
@atulbhatt-system32 
did you add this line cuz you got errors? of course sometimes I got errors while undo/redo.

for the current sync logic that I've completed, the function should has no error I guess. I mean after fixing node-path issue!

so I did left that to check if the sync works correctly or not in the future.

Reply:
I was opening .md file in that case I got it. As we haven't yet completed the other file opening issue.

Reply:
No worries, even if we are putting this condition. If something won't be working we will see it not working.

Reply:
got it.